pylint/checkers/variables.py

```diff
@@ -2237,9 +2237,12 @@ def _loopvar_name(self, node: astroid.Name) -> None:
             if (
                 isinstance(inferred, astroid.Instance)
                 and inferred.qname() == "builtins.enumerate"
-                and assign.iter.args
             ):
-                inferred = next(assign.iter.args[0].infer())
+                likely_call = assign.iter
```

Comment:
Coincidentally, this could be done with a ternary 😄 

But we never use those!

Reply:
I feel like Daniel wouldn't be a fan of this checker : https://github.com/PyCQA/pylint/pull/4871 😉